tooling/cli/src/interface/rust.rs

```diff
@@ -228,8 +228,16 @@ impl Interface for Rust {
       self.app_settings.target_triple.clone(),
     );
 
-    let mut s = self.app_settings.target_triple.split('-');
-    let (arch, _, host) = (s.next().unwrap(), s.next().unwrap(), s.next().unwrap());
+    let target_triple = self.app_settings.target_triple.clone();
```

Comment:
I think you shouldn't need to clone here
```suggestion
    let target_triple = self.app_settings.target_triple;
```